Docker/Dockerfile

```diff
@@ -6,7 +6,7 @@ SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 ARG DEBIAN_FRONTEND=noninteractive
 RUN apt-get update && \
 	apt-get install --no-install-recommends -y \
-	ca-certificates cron \
+	ca-certificates cron libapache2-mod-auth-openidc \
```

Comment:
Our other Dockerfiles should be updated as well

Reply:
It seems that this module doesn't have a package for alpine. I can add a note in the documentation.

Reply:
https://pkgs.alpinelinux.org/packages?name=apache-mod-auth-openidc&branch=edge&repo=&arch=x86_64

Reply:
Ah, I see. This package can only be found in edge. I updated Dockerfile-Newest. Older alpines do not have this package.